ergotree-ir/src/mir/header.rs

```diff
@@ -1,3 +1,5 @@
+use num_bigint::BigInt;
```

Comment:
`mir` (Middle Intermediate Representation) module is intended for IR nodes. New `chain` module for types specific to the blockchain seems like a good fit.